radix_sort/Java/radix_sort.java

```diff
@@ -0,0 +1,40 @@
+package com.javatutorial;
+  
+import java.util.ArrayList;
+import java.util.List;
+  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp75peg948/radix_sort/Java/radix_sort.java
+++ b/tmp/tmp75peg948/radix_sort/Java/radix_sort.java
@@ -2,7 +2,7 @@
   
 import java.util.ArrayList;
 import java.util.List;
-  
+
 public class ExampleRadixSort {
     public static void main(String[] args) {
         int[] num = {170, 45, 75, 90, 802, 2, 24, 66,23,234,3,232,44};
```